test/e2e/lib/pages/gutenboarding-page.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { By as by } from 'selenium-webdriver';
+
+/**
+ * Internal dependencies
+ */
+import AsyncBaseContainer from '../async-base-container';
+import * as dataHelper from '../data-helper';
+import * as driverHelper from '../driver-helper';
+
+export default class GutenboardingPage extends AsyncBaseContainer {
+	constructor( driver, url ) {
+		if ( ! url ) {
+			url = GutenboardingPage.getGutenboardingURL();
+		}
+		super( driver, by.css( '.is-section-gutenboarding' ), url );
+	}
+
+	static getGutenboardingURL() {
+		// using forceWpCalypso flag because Gutenboarding is enabled only on wpcalypso
+		return dataHelper.getCalypsoURL( 'gutenboarding', undefined, { forceWpCalypso: true } );
+	}
+
+	async waitForBlock() {
+		return await driverHelper.isElementPresent(
```

Comment:
Does the block always show up right away? I generally try to avoid the `isElementPresent` function unless I am sure that it will be there or I've checked to make sure that something relative to it is present already. You may want to consider using `isEventuallyPresentAndDisplayed` if you see any issues with timing.

Reply:
Right now we show the block right away but giving it a second look, I see the point in using `isElementPresent` for checking elements after initial render.